test.ts

```diff
@@ -129,6 +129,42 @@ Deno.test("main", async (t) => {
     await relay.shutdown();
 });
 
+// https://github.com/nostr-protocol/nips/blob/master/11.md
+Deno.test("NIP-11: Relay Information Document", async (t) => {
+    try {
+        await Deno.remove("test.sqlite");
+    } catch (e) {}
+    const relay = await run({
+        password: "123",
+        port: 8080,
+        default_policy: {
+            allowed_kinds: "none",
+        },
+        default_information: {
+            name: "Nostr Relay",
+        },
+        kv: await Deno.openKv("test.sqlite"),
+    }) as Relay;
+
+    await t.step("get relay information", async () => {
+        const information = await relay.get_relay_information();
+        console.log(`information`, information);
+
+        assertEquals(information.name, "Nostr Relay");
```

Comment:
let's assert the whole inforamtion object.

Reply:
ok